src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -260,6 +260,10 @@
     type: int
     short-summary: Load balancer idle timeout in minutes.
     long-summary: Desired idle timeout for load balancer outbound flows, default is 30 minutes. Please specify a value in the range of [4, 120].
+  - name: --outbound-type
```

Comment:
there's a PR in extension repo revert the behavior, could you pls sync: https://github.com/Azure/azure-cli-extensions/pull/1405? @gtracer

Reply:
@yungezz I own both PRs. 

In extension, outbound-type is already enabled. That PR just removes one piece of validation (network-plugin) and adds a vnetsubnet-id validation. No core behavior is being reverted. Please let me know what you think is specifically being reverted

Reply:
@yungezz following up as I need to merge this PR before the cutoff for the release